generators/server/templates/src/routes/api/v1/usersRouter.js

```diff
@@ -0,0 +1,21 @@
+import express from "express";
+import passport from "passport";
+import { User } from "../../../models/index.js";
+
+const usersRouter = new express.Router();
+
+usersRouter.post("/", async (req, res) => {
+  const { email, password, passwordConfirmation } = req.body;
+  const user = new User({ email, password });
+  try {
+    const persistedUser = await User.query().insertAndFetch({ email, password });
+    return req.login(persistedUser, () => {
+      return res.status(201).json({ user: persistedUser });
+    });
+  } catch (error) {
+    console.log(error);
+    return res.status(422).json({ errors: error });
+  }
+});
```

Comment:
Our current pattern in curriculum is to check for a validation error here

additionally, we also set status of 500 if it isnt a validation error

Pattern in Staff GP app 
https://github.com/LaunchAcademy/coding-crossroads/blob/master/server/src/routes/api/v1/usersRouter.js

Reply:
this is not a pattern we want to perpetuate. A 500 should bubble to a higher order middleware that appropriately renders an error page.